webapp/src/lib/account/create.ts

```diff
@@ -126,7 +126,9 @@ export default async function createAccount({
 					storage: 0
 				},
 				stripeTrial: false
-			}
+			},
+			//SET ONBOARDED TO TRUE FOR NOW
```

Comment:
Remove any temp values

Reply:
Also this will make invited team members go to onboarding. Just confirm with Misri+Andrew what the behaviour should be, but I don't think invited members should be sent to the onboarding screen.

Reply:
added a role permission check. if role has permission to create models then onboarded is set to false